sdk/identity/Azure.Identity/src/Azure.Identity.csproj

```diff
@@ -2,9 +2,9 @@
   <PropertyGroup>
     <Description>This is the implementation of the Azure SDK Client Library for Azure Identity</Description>
     <AssemblyTitle>Microsoft Azure.Identity Component</AssemblyTitle>
-    <Version>1.8.0</Version>
+    <Version>1.8.1</Version>
     <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
-    <ApiCompatVersion>1.7.0</ApiCompatVersion>
```

Comment:
I didn't catch this (lack of) change. It's concerning the bot didn't update this before. Was this branched from the 1.7.0 commit, can I safely assume?